CONTRIBUTING.mdx

```diff
@@ -42,7 +42,7 @@ We use a bunch of tools to help the development process and to ease maintenance.
 
 ### Linters
 
-We use `eslint` and `stylelint` to check both the JS code and the CSS-in-JS. Linters run as a `jest-runner`, which provides a nice developer experience like jest watch mode.
```

Comment:
Unfortunately `stylelint` doesn't really work well with CSS-in-JS, which is what we use here.
Since we don't use normal CSS files, we can remove the tool.